src/ProductlineApp.WebUI/ClientApp/src/components/molecules/settingsSections/changePersonalDataSection/ChangePersonalDataSection.tsx

```diff
@@ -1,14 +1,45 @@
+import { useState } from 'react';
+import ChangePersonalDataButton from '../../../atoms/buttons/changePersonalDataButton/ChangePersonalDataButton';
 import EmailInput from '../../../atoms/inputs/emailInput/EmailInput';
 import UsernameInput from '../../../atoms/inputs/usernameInput/UsernameInput';
 import './css/changePersonalDataSection.css';
+import ConfirmChangePersonalDataButton from '../../../atoms/buttons/confirmChangePersonalDataButton/ConfirmAccountDataButton';
+import CancelButton from '../../../atoms/buttons/cancelButton/CancelButton';
+import ButtonsSection from '../../../molecules/settingsSections/buttonsSection/ButtonsSection';
 
 function ChangePersonalDataSection() {
+
+  const [disableEdit, setDisableEdit] = useState(true)
+
   return (
     <>
-      <div className="changePesonalDataSection">
-        <UsernameInput />
-        <EmailInput />
-      </div>
+      <form>
+        <div className="changePesonalDataSection">
+          <div className="userNameField">
+            <label htmlFor="uname" className="unameLabel">Nazwa uzytkownika</label>
+            <input
+              type="text"
+              id="uname"
+              name="uname"
+              placeholder="Nazwa uzytkownika"
+              className="changeUsernameInput"
+              disabled={disableEdit}
+            ></input>
+          </div>
+          <div className="emailInputField">
+            <label htmlFor="email" className="emailLabel">Email</label>
+            <input
```

Comment:
to samo - jesli sie da, to uzywaj komponentu FormInput (zwlaszcza gdy pole jest czescia formularza)